content/millennium/r4/general-clinical/condition.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Condition | R4 API
+---
+
+# Condition
+
+* TOC
+{:toc}
+
+## Overview
+
+The Condition resource is used to record problems, diagnoses, or other health matters that are concerning to a patient. Depending on the site, problems and diagnoses are used in different ways. Diagnoses are recorded on an encounter to document conditions addressed or treated during a specific patient visit. For U.S. clients, diagnoses support financial or reimbursement workflows. Problems are on-going or chronic conditions that are managed over every visit and not associated to a specific encounter. A condition may be both a diagnosis and a problem. Conditions, specifically diagnoses, are often associated to orders to document the medical necessity of a specific medication or procedure order and may be referenced by other resources as a “reason” for an action or order. Health concerns are also returned as part of the Condition resource and represent other concerns a patient may have such as financial or social risks.
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id)
```

Comment:
shouldn't we be pointing all of these to the https:// urls?

Reply:
Fixed. Throughout. [4872fd1]